include/wasp/convert/to_binary.h

```diff
@@ -23,16 +23,22 @@
 
 #include "wasp/base/at.h"
 #include "wasp/base/buffer.h"
+#include "wasp/base/features.h"
 #include "wasp/base/optional.h"
 #include "wasp/binary/types.h"
 #include "wasp/text/types.h"
 
 namespace wasp::convert {
 
 struct Context {
+  explicit Context() = default;
```

Comment:
What does explicit mean for no-args constructors?

Reply:
It means you can't automatically convert it from `{}`, e.g.: https://godbolt.org/z/qKW8d6

Reply:
Is that useful/important in this case?   If no-arg construction possible why not allow it?